### DISCUSSION TITLE
滚轮快切具体是指什么功能

### DISCUSSION THREAD

**Discussion by xmbai9507** [shuax/MouseInc.Settings#492] [Q&A]
<img width="322" alt="image" src="https://github.com/shuax/MouseInc.Settings/assets/65625632/20f69f5d-7e8b-4f9c-9790-27e4c78f9e94">

我理解是, 可以对同一手势(比如-->)设置不同功能(比如前进以及后退), 然后向右划选触发之时能通过滚轮切换到底是执行前进还是执行后退.
但试了一下发现不这样, 似乎是我理解有误, 所以请问这个功能具体指的是什么?

**Comment by Amppa** [comment]
同問+1
